CHANGELOG.md

```diff
@@ -8,6 +8,10 @@
 
 * Issue #360 : write a dataset with a specified filename do not open a dialog except if the file already exists.
 
+* read_dir: now ignores non readable files
```

Comment:
and read!  which also do directory reading (see example in test_importer)

Reply:
done